.circleci/config.yml

```diff
@@ -1,12 +1,16 @@
 version: 2.1
 
 orbs:
-  sixrs-gke: sixriversystems/sixrs-gke@3
+  sixrs-gke: sixriversystems/sixrs-gke@7
+
+aliases:
+  - &node_image cimg/node:16.19.0
+  # - &npm_global_cache dependency-cache-v4-{{ checksum ".node-version" }}-{{ checksum "package-lock.json" }}
```

Comment:
should this be removed?

Reply:
I don't think it needs to be, it's part of the "template" that we might want to use here in the future